### ISSUE TITLE
Is clientjs an idled project?

### ISSUE THREAD

**Issue by mohaa7** [jackspirou/clientjs#155]
Is it abandoned?

As the code has not been touched for a year, is it recommended to use in production, in terms of bug reports followup?

Will it have any future major release?

**Comment by avpaderno** [comment]
If you are looking for a different library, there is the [FingerprintJS](https://github.com/fingerprintjs/fingerprintjs) library, which has an accuracy between 40% and 60%, but it is fully open source.
ClientJS itself has been built upon [Valve/fingerprintjs](https://web.archive.org/web/20200714191004/https://github.com/Valve/fingerprintjs), which then evolved into fingerprintjs/fingerprintjs.

**Comment by jackspirou** [comment]
ClientJS is not built on FingerprintJS, but you should def use FingerprintJS! It's professionally maintained, and does provide good accuracy for an open source, client side only product.

**Comment by avpaderno** [comment]
I apologize: I was "quoting" the project page, which says *Built Upon:* and shows three links, including &nbsp;`https://web.archive.org/web/20200714191004/https://github.com/Valve/fingerprintjs`. 

I did not mean this library has code that comes from that library, but that the project page has links to other libraries, which I take it as meaning *These are other libraries with a similar purpose.* If I were to look to alternatives, I would check those three links.

**Comment by acastro2** [comment]
https://github.com/fingerprintjs/fingerprintjs just change its license so it can't run in production anymore

**Comment by avpaderno** [comment]
3.x and lower versions are still licensed under MIT.

**Comment by peter-wd-1** [comment]
> 3.x and lower versions are still licensed under MIT.

is this true? I can use it on production?

**Comment by mohaa7** [comment]
> I can use it on production?

Changes are not retroactive. The code in version4 is still readble, but not open source anymore! By version3 you will be stuck on an immobile creature!

**Comment by avpaderno** [comment]
Yes, Branch 3.x is still licensed under MIT. I guess they will not back-port to branch 3.x the changes they introduce in branch 4.x, though.
